MbDotNet/MountebankRequestProxy.cs

```diff
@@ -121,6 +123,24 @@ private HttpResponseMessage ExecutePost(string resource, string json)
             }
         }
 
+        private void HandleDynamicPort(HttpResponseMessage response, Imposter imposter)
+        {
+            if (imposter.Port == null)
+            {
+                try
+                {
+                    var content = response.Content.ReadAsStringAsync().Result;
+                    Match match = new Regex(".*\"port\": (\\d+),", RegexOptions.Multiline).Match(content);

```

Comment:
I with there was a better way to do this than just parsing through the response content. My first thought was to deserialize the JSON to a `RetrievedImposter` type like the `GetImposter` method does. Technically, we could since we have the original imposter object so we would know what protocol it is, but that feels weird as well. I think I am fine with this for now. I can always update it later if I think of a better way.